### PR TITLE
feat: Implement updated ui for dashboard view 

### LINKED ISSUE #148: Implement Dashboard updated ui

### PR DESCRIPTION
**Pull request by talwinder50**
https://user-images.githubusercontent.com/7862595/210935997-8fc8c8a5-8c70-4097-850d-2d3755a0b271.MP4

closes #148 
TODO: Credential list will be updated in the followup task

### CODE REVIEW

File: demo/app/lib/views/credential_list.dart [hunk 1]
```diff
@@ -45,28 +44,40 @@ class _CredentialListState extends State<CredentialList> {
...
+                child: const Text(
+                  'Credentials',
+                  style: TextStyle(color: Color(0xff190C21), fontWeight: FontWeight.bold, fontSize: 16),
+                )),
+            Container(
+                alignment: Alignment.center,
+                child: _loading
+                  ? const CircularProgressIndicator()
+                  : _credentialList.isEmpty
+                  ? const Text("No Credentials found")
```
Comment (DRK3) [inline demo/app/lib/views/credential_list.dart:63]: `Credentials` should be lowercase

File: demo/app/lib/views/credential_preview.dart [hunk 2]
```diff
@@ -47,7 +47,7 @@ class CredentialPreviewState extends State<CredentialPreview> {
         shrinkWrap: true,
         itemBuilder: (context, position) {
           //TODO Ignoring the photo value for now due to extremely long text need to render in a separate issue-881
```
Comment (DRK3) [inline demo/app/lib/views/credential_preview.dart:49]: (not directly related to this, but where does this issue# come from? I can't find it)

Comment (DRK3) [inline demo/app/lib/views/credential_preview.dart:49]: (not directly related to this, but where does this issue# come from? I can't find it)

**Review by DRK3 — APPROVED** [review]
Minor comments only

File: demo/app/lib/views/credential_preview.dart (hunk 2 above)
Comment (talwinder50) [reply demo/app/lib/views/credential_preview.dart:49]: was typo and seems like original issue has been closed , i have created a separate task and updated the issue number here

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**